core/block_svg.js

```diff
@@ -639,6 +639,20 @@ Blockly.BlockSvg.prototype.onMouseDown_ = function(e) {
     // Click, not drag, so stop waiting for other touches from this identifier.
     Blockly.Touch.clearTouchIdentifier();
   } else if (!this.isMovable()) {
+    // Click handler for immovable blocks.
+    var mouseupWrapper = Blockly.bindEvent_(document,
```

Comment:
> This should be bindEventWithChecks_.

I was using bindEventWithChecks_ and it took me a couple of hours to figure out that that was why the mouseup event was never getting executed.  Switching to bindEvent_ got the event to work.

> Have you checked that this works on touch, especially with multi-touch?

I'm unable to check touch, since you guys have my iPad.  I'm also less familiar with Blockly's new touch event strategy (which is why it took hours to figure out that WithChecks was eating the event).  Can you take this over?